app/models/text_field_type.rb

```diff
@@ -20,6 +22,22 @@ def mapping
 
   private
 
+  def less_than
+     errors.add(:text, "must be less_than #{@validations[:max]}") if @integer <= validations[:max]
+  end
+
+  def greater_than
+     errors.add(:text, "must be greater_than #{@validations[:min]}") if @integer >= validations[:min]
+  end
+
+  def validate_max?
+    @validations.key? :max
```

Comment:
So I get the idea behind using the text field type and giving it a numeric input since they share many consistencies, but these validations would only ever exist if the render_method is numeric, which starts to tie up validation logic with our view logic. Though it may seem redundant, I would recommend building out a separate, simple, integer FieldType to avoid this.